test-requirements.txt

```diff
@@ -1,6 +1,8 @@
 codecov
 coverage
 flake8; python_version >= '2.7'
-mock
+mock==2.0.0; python_version == '2.6'
+mock==3.0.5; python_version >= '2.7' and python_version <= '3.5'
+mock==4.0.2; python_version >= '3.6'
```

Comment:
>mock==4.0.2; python_version >= '3.6'

Now would the mock version always be fixed in Travis too? Like if 5.* is released then would travis still run 4.0.2?

Reply:
Yeah, it would. But we can't leave the version undefined because it will then default to the pre-installed one which is 2.0.0.

Reply:
dang :/